src/frontend/src/store/paginator/index.js

```diff
@@ -18,5 +18,5 @@ export const paginator = (items, page, per_page) => {
   console.log("Paginate Item: ", paginatedItems);
   console.log("Pagin Content: ", paginContent);
   const total = paginContent.total;
-  return paginContent.data;
-};
\ No newline at end of file
+  return paginContent;
```

Comment:
This is the main change. Instead of discarding the `paginContent`, we need to keep it around to have access to the number of pages.

Reply:
You are correct, the whole object had to be returned. When you pass only the data I do not have access to the total pages.

That was a stupid mistake on my part

Reply:
Definitely not stupid. To build a complete application like RyuAnime, it's easy to mistake one value for another 👍 and it took me quite some time to realize what was going on. Thanks for review/merging it so quickly! Now responsive layout is the next one.